docs/non-normative/k8s-naming.md

```diff
@@ -0,0 +1,22 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: K8s naming
+--->
+
+# Kubernetes Naming Guidance
+
+When naming resource, attributes, or metrics related to Kubernetes objects, **prioritize consistency with Kubernetes API naming** over certain Opentelemetry's naming conventions, as long as the resulting name is valid per the general naming rules.
```

Comment:
```suggestion
When defining resource, attributes, or metrics related to Kubernetes objects, use names that are consistent and align with the Kubernetes API naming.
```

OR

```suggestion
When naming resources, attributes, or metrics related to Kubernetes objects, prioritize consistency with Kubernetes API naming over the [general naming consideration](docs/general/naming.md) defined by semantic conventions, as long as the resulting name is valid per the general naming rules.
```

(please check the link before accepting, did not test it)

Reply:
The latest PR has the link. Resolving this comment. Lmk if you'd like to see any more edits.